newIDE/app/src/EventsSheet/EventsTree/Renderers/LinkEvent.js

```diff
@@ -78,43 +80,51 @@ export default class LinkEvent extends React.Component<EventRendererProps, *> {
         className={classNames({
           [largeSelectableArea]: true,
           [largeSelectedArea]: this.props.selected,
+          [linkContainer]: true,
         })}
         style={styles.container}
       >
-        <span style={styles.title}>
-          Include events from{' '}
-          <i
+        <span
             className={classNames({
               [selectableArea]: true,
-            })}
-            onClick={this.edit}
+              [disabledText]: this.props.disabled,
+            })}            
+        >
+          <span style={styles.title}>
```

Comment:
Here:

```
<span 
  style={styles.title}
  className={classNames({
    [disabledText]: this.props.disabled,
  })}
>
```

Using this, we only add the "disabled" to the title. It's simpler and avoid risking of polluting InlinePopover and ExternalEventsField with extra styling that could break down the rendering of these components.